frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.js

```diff
@@ -17,19 +17,17 @@ class CrossingListHeader extends Component {
     super(props);
     this.state = {
       showFilterDrawer: false,
-      invertSort: false
     };
   }
 
   toggleFilterDropdown = () => { this.setState({ showFilterDrawer: !this.state.showFilterDrawer }) };
-  toggleSortDirection = () => { this.setState({ invertSort: !this.state.invertSort }) };
 
   render() {
     if ( !this.props.data || this.props.data.loading) {
       return '';
     };
 
-    const { toggleShowOpen, toggleShowClosed, toggleShowCaution, toggleShowLongterm, showOpen, showClosed, showCaution, showLongterm } = this.props;
+    const { toggleShowOpen, toggleShowClosed, toggleShowCaution, toggleShowLongterm, toggleSort, showOpen, showClosed, showCaution, showLongterm, invertSort } = this.props;
```

Comment:
minor, but can we call invertSort something like sortByUpdatedAsc in case we sort by something else later like Status. 
Similarly, toggleSortByUpdatedAsc.